_pages/features.md

```diff
@@ -258,53 +258,44 @@ Note the global `$script` and `$runner` variables: reach any part of atoum to fi
 ## Third-party integration
 
 atoum integrates well with those CI tools:
-<ul class="columns integrations" data-columns="5">
+<ul class="autocolumns integrations">
     <li>
-       <img src="images/integrations_logos/travis-ci.svg" />
-       <a href="https://travis-ci.org/">Travis CI</a>
+       <a href="https://travis-ci.org/"><img src="images/integrations_logos/travis-ci.svg" alt="" />Travis CI</a>
```

Comment:
putting empty alt is bit sad... Could you fill them?

Reply:
Use `role="presentation"` with no `alt` in this case, it's perfect.

Reply:
Empty @alt are perfectly legible...

They are required by the HTML specifications, and must be filled only when this is relevant. Since those are just logos, and there is already a describing text in the links; empty alt are ok. Filling them would decrease readability if images are not displayed :)

Reply:
I'm not sure about the role attribute... Why would we add semantic on those non significative elements?

Reply:
for me it's ok to let it like that, or to put the role whataver